pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -20,15 +21,17 @@ import (
 )
 
 const (
-	functionNameEnvVar = "AWS_LAMBDA_FUNCTION_NAME"
+	functionNameEnvVar         = "AWS_LAMBDA_FUNCTION_NAME"
+	captureLambdaPayloadEnvVar = "DD_CAPTURE_LAMBDA_PAYLOAD"
```

Comment:
Why not use the configuration for this ?

Reply:
captureLambdaPayloadEnvVar has been moved to config